hardly/handlers/distgit.py

```diff
@@ -31,14 +31,46 @@ def __init__(
             job_config=job_config,
             event=event,
         )
+        self.mr_identifier = event.get("identifier")
+        self.mr_title = event.get("title")
+        self.mr_description = event.get("description")
+        # Gitlab specific, but hardly is too
+        self.mr_url = (
+            f"{event.get('project_url')}/-/merge_requests/{self.mr_identifier}"
```

Comment:
If you look at the MR webhook in test data in service, there is an `url` field in `event`. Or there is a problem with that?

Reply:
No problem, I was just lazy to create a PR in p-s :-)
Here it is packit/packit-service#1197